init/src/hpx_wrap.cpp

```diff
@@ -25,8 +25,8 @@ namespace hpx_start
     // by this code.
     HPX_SYMBOL_EXPORT extern bool include_libhpx_wrap;
     HPX_SYMBOL_EXPORT bool include_libhpx_wrap __attribute__((weak)) = false;
-    HPX_SYMBOL_EXPORT extern std::string app_name_libhpx_wrap
-        __attribute__((weak));
```

Comment:
extern and weak usually don't go together in macOS (see #3427). That's a good catch.